efficientdet/efficientdet_arch.py

```diff
@@ -282,6 +282,108 @@ def class_net(images,
+                                        padding='same',
+                                        name='class-%d' % i))
+
+            # Level only apply here so it's maybe better inside (no need to use tf.AUTO_REUSE anymore)
+            bn_act_ops_per_level = {}
+            for level in range(config.min_level, config.max_level + 1):
+                bn_act_ops_per_level[level] = utils.Batch_norm_act(is_training,
+                                                                   act_type=act_type,
+                                                                   init_zero=False,
+                                                                   use_tpu=use_tpu,
+                                                                   data_format=data_format,
+                                                                   name='class-%d-bn-%d' % (i, level))
+            self.bn_act_ops.append(bn_act_ops_per_level)
+
+        if self.use_dc:
+            self.dc = utils.Drop_connect(survival_prob)
+
+        if separable_conv:
+            self.classes = tf.keras.layers.SeparableConv2D(
+                                            filters=num_classes * num_anchors,
+                                            depth_multiplier=1,
+                                            pointwise_initializer=tf.initializers.variance_scaling(),
+                                            depthwise_initializer=tf.initializers.variance_scaling(),
+                                            data_format=data_format,
+                                            kernel_size=3,
+                                            activation=None,
+                                            bias_initializer=tf.zeros_initializer(),
+                                            padding='same',
+                                            name='class-predict')
+
+        else:
+            self.classes = tf.keras.layers.Conv2D(
+                                            filters=num_classes * num_anchors,
+                                            kernel_initializer=tf.random_normal_initializer(stddev=0.01),
+                                            data_format=data_format,
+                                            kernel_size=3,
+                                            activation=None,
+                                            bias_initializer=tf.zeros_initializer(),
+                                            padding='same',
+                                            name='class-predict')
+
+    def call(self, feats, level=None, **kwargs):
+        image = feats
+        for i in range(self.repeats):
+            original_image = image
+            image = self.conv_ops[i](image)
+            image = self.bn_act_ops[i][level].call(image)
+            if i > 0 and self.use_dc:
+                image = self.dc.call(image)
+                image = image + original_image
+
+        return self.classes(image)
+
+
```

Comment:
This class is missing a `get_config` method.

One of the great things about keras layers is that they can be serialized for saving and loading.

`get_config` should be defined so that this works.

```
config = existing_layer.get_config()
new_layer = ClassNet(**config)
```